modules/dev-tools/assets/js/module.js

```diff
@@ -0,0 +1,27 @@
+import Deprecation from './deprecation';
+import { consoleWarn } from './utils';
+import { softDeprecated } from '../js/deprecation/utils';
+
+/* global elementorDevToolsConfig */
+
+export default class Module {
+	constructor() {
+		this.deprecation = new Deprecation();
+
+		this.notifyBackendDeprecations();
+	}
+
+	notifyBackendDeprecations() {
+		// eslint-disable-next-line camelcase
+		const notices = elementorDevToolsConfig.deprecation.soft_notices;
+
+		Object.entries( notices ).forEach( ( [ key, notice ] ) => {
+			softDeprecated( key, ...notice );
```

Comment:
Why soft? Use `this.deprecation.deprecated`

Reply:
Handled.